src/Pages/Menu/Goods.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+
+class Goods extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isHovering: "none",
+      isClicked: false,
+    };
+  }
+
+  handleMouseOver = () => {
+    this.setState({ isHovering: "block" });
+  };
+
+  handleMouseLeave = () => {
+    this.setState({ isHovering: "none" });
+  };
+
+  handleHeartClick = () => {
+    this.setState((prevState) => ({
+      isClicked: !prevState.isClicked,
+    }));
+  };
+
+  render() {
+    // console.log(this.props.category);
+    return (
+      <li>
+        <div
+          className="goods-block"
+          onMouseEnter={this.handleMouseOver}
+          onMouseLeave={this.handleMouseLeave}
+        >
+          <div className="goods-icon">
+            <em className={this.props.best ? "best" : "new"}>
+              {this.props.best ? "best" : "" || this.props.new ? "new" : ""}
+            </em>
+          </div>
+          <div className="goods-thumb">
+            <p>
+              <a href="/#">
+                <img
+                  alt="goodsDetail"
+                  src={this.props.img}
+                  className="goods-detail-view"
+                />
+              </a>
+            </p>
+            <div
+              onClick={this.handleHeartClick}
+              className={
+                this.state.isClicked
+                  ? "goods-opt-selectbox-fill"
+                  : "goods-opt-selectbox"

```

Comment:
```suggestion
                  ? "goods-opt-selectbox fill"
                  : "goods-opt-selectbox"
```
이렇게 분리하면 공통되는 스타일과 변화하는 스타일 관리하기 더 좋을 거 같습니다